docs/book/service-manager.md

```diff
@@ -46,7 +46,7 @@ return [
             'writers' => [
                 'stream' => [
                     'name' => 'stream',
-                    'priority' => \Zend\Log\Logger::ALERT,
+                    'priority' => 1,
```

Comment:
I fail to see why this is a good change.

The `Zend\Log\Logger` constants directly correspond to syslog message severities as described in [IETF RFC 3164 Section 4.1.1](https://tools.ietf.org/html/rfc3164#section-4.1.1). While we use them internally with the priority filter, they are also used with the various writers to determine how to emit log messages.

This particular constant... evaluates to `1`. Which is the default priority we want to use. The constant is far more descriptive than an integer value here. As such, I cannot accept this PR at this time.

Reply:
Ok, I did not understand the principle.
I understood that "priority" is the loading order of "writers".
I read that the bigger the number, the higher the priority.
DEBUG (7) is bigger than EMERG (0) but debug is less important than emerg, you see ?

Reply:
Ugh! You're absolutely correct!

I dug into where this value is used, and the `LoggerAbstractFactory` takes that value and uses it to call `Logger::addWriter()`. The value is then used to seed an `SplPriorityQueue`, which determines the order in which writers are invoked.

So... re-opening!